spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,34 @@
+require './lib/facility_factory'
+require './lib/dmv_data_service'
+require './lib/facility'
+
+RSpec.describe FacilityFactory do
+
+  it 'exists' do
+    factory = FacilityFactory.new
+    expect(factory).to be_a(FacilityFactory)
+  end
+
+  it 'creates Facilities' do
+    factory = FacilityFactory.new
+
+    or_dmv_facilities = DmvDataService.new.or_dmv_office_locations
+    or_facilities = factory.create_facilities(or_dmv_facilities)
+
+    expect(or_facilities).to be_an(Array)
+    expect(or_facilities[0]).to be_a(Facility)
```

Comment:
Yup, great job with this testing. We can't test the content because that could change since we don't own or control that data.